superset/assets/src/components/TableSelector.jsx

```diff
@@ -43,6 +43,7 @@ const propTypes = {
   onChange: PropTypes.func,
   clearable: PropTypes.bool,
   handleError: PropTypes.func.isRequired,
+  forceDisableTableSelector: PropTypes.bool,
```

Comment:
why `forceDisable`? I'd just call this `disableTableSelector`

Reply:
In function `renderTable()`, there is an existed logic to disable Select, which is based on database attribute or schema state. `forceDisableTableSelector` indicates this logic is  outside component itself.